tf2_web_republisher_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.22)
+project(tf2_web_republisher_msgs)
```

Comment:
No we are splitting msgs from the library for the ROS2 port. Also stick to ROS2 standards. Which means using X_interfaces instead of X_msgs

Reply:
+1 for using `X_interfaces`